load/loader.go

```diff
@@ -102,6 +113,15 @@ func GetBenchmarkRunner(c BenchmarkRunnerConfig) BenchmarkRunner {
 			panic(fmt.Sprintf("could not initialize BenchmarkRunner: %v", err))
 		}
 	}
+
+	if c.ReportingMetricsPort > 0 {
```

Comment:
Since this type is uint64 and thus always positive or `0`, you could potentially drop this condition. Doing so would require sanitizing value once and would reduce cyclomatic complexity by removing multiple `if`s used with this variable.

Reply:
I'd love to drop the if condition.

> (but still need to sanitize it when variable is read from CLI flag)

I'm not sure what you mean by sanitizing here. As we have this config that allows using json, yaml or params from CLI, we have the default value that is the real port. I used 0 in the tests to ignore the metrics.

I can move the rules to `pushEventToPrometheus` but I'm not that familiar with go, so, if we have a way o get ride of the if at all, I'm all for the change!